.github/workflows/workflow.yaml

```
@@ -0,0 +1,35 @@
+name: build_test
```

```suggestion
name: test
```

we are not building yet, let's just change to test